data/assets/scene/solarsystem/planets/saturn/minor/norse_group.asset

```diff
@@ -433,25 +433,24 @@ local norseGroup = {
     },
     TrailColor = trailColor,
     OrbitPeriod = 1212.53,
-    Kernels = kernel368
-  },
-  -- Not yet in the Spice kernels
-  -- {
-  --   Identifier = "Sutur", -- Rising
-  --   Parent = {
-  --     Identifier = parentIdentifier,
-  --     Spice = parentSpice
-  --   },
-  --   Spice = "SUTUR",
-  --   Radii = { 6000, 6000, 6000 },
-  --   Tags = tags,
-  --   GUI = {
-  --     Path = "/Solar System/Planets/Saturn/Moons/Norse Group"
-  --   },
-  --   TrailColor = trailColor,
-  --   OrbitPeriod = 1242.36,
-  --   Kernels = kernels368
-  -- },
+    Kernels = kernel452
+  },
+  {
+    Identifier = "Surtur", -- Rising
```

Comment:
Is this comment literally just here as a reference to an Amon Amaranth album? ;)